periodic-tasks/check-for-firmware-updates/io.elementary.settings-daemon.check-for-firmware-updates.service

```diff
@@ -0,0 +1,8 @@
+[Unit]
+Description=Check for firmware updates daily
+ConditionACPower=true
+After=network.target network-online.target systemd-networkd.service NetworkManager.service connman.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/libexec/io.elementary.settings-daemon.check-for-firmware-updates
```

Comment:
what if, instead of an helper application, use an "check-firmware-updates" Action in the daemon and them use `busctl (or any other terminal dbus command, like gdbus) --user call io.elementary.settings-daemon /io/elementary/settings_daemon org.freedesktop.Application ActivateAction sava{sv} "app.check-firmware-updates" 0 0`. that should solve the issue about the ability to activate the notification.

Reply:
@Marukesu thank you very much for your feedback! This now works as expected.